app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedTopics.kt

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * <h1>POJO class to parse related topics object in retrofit response from websearch client.</h1>
+
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics
+
+(@SerializedName("FirstURL")
+ @Expose
+ var url: String?,
+
+ @SerializedName("Text")
+ @Expose
+ var text: String?,
+
+ @SerializedName("Icon")
+ @Expose
+ var icon: WebIcon?) {
+
+    @SerializedName("Result")
+    @Expose
+    var result: String? = null
+}
```

Comment:
Remove from them too. There are 16 POJO classes in response package. Out of which only 4 has a constructor: WebIcon, WebSearch, RelatedTopics and SusiBaseUrls. Out of which, constructor in SusiBaseUrls is actually getting used whereas rest three have useless constructor and coincidently those three POJO classes ( WebIcon, WebSearch, RelatedTopics ) are made by you only. And suppose lets say you may even need to initialize the variables, then why would you write a constructor to initialize 3 out of 4 variables. What has 4th variable done to you? Please, let's end the discussion. Just like other 12 POJO classes, please don't write a constructor in these 3 until and unless it is actually getting used like SusiBaseUrls.